app/src/main/res/layout/fragment_add_product_ingredients.xml

```diff
@@ -121,7 +121,7 @@
             app:layout_constraintTop_toBottomOf="@id/hint_ingredients_list" />
 
         <Button
-            android:drawableLeft=" @drawable/ic_file_download_black_18dp"
+            android:drawableLeft="@drawable/ic_file_download_black_18dp"
```

Comment:
actually, this icon doesn't work at all. The old compare one doesn't cause the crash.